kanidm_unix_int/src/cache.rs

```diff
@@ -645,7 +666,11 @@ impl CacheLayer {
                     name: self.token_uidattr(&tok),
                     gid: tok.gidnumber,
                     gecos: tok.displayname,
-                    shell: tok.shell.unwrap_or_else(|| self.default_shell.clone()),
+                    // WIP #392: default shell
+                    // this should now be valid, so simply unwrap it?
+                    // where and how is `self.default_shell` set?
+                    shell: tok.shell.unwrap_or_default(),
```

Comment:
This line needs to be removed. There may be old, cached information in the sqlite store for accounts where the account is None, and this would cause a panic when the update is deployed, so the unwrap_or_else logic is still requried here as a safety barrier.

Reply:
Good point. I will revert.